src/main/kotlin/no/digdir/fdk/statistics/repository/StatisticsRepository.kt

```diff
@@ -0,0 +1,56 @@
+package no.digdir.fdk.statistics.repository
```

Comment:
Har du satt opp riktig indekser på alle columns man bruker i where clause?

Reply:
86a94faf78bb4d7903d34c04b52dbd71e346b8ea
Noe mer enn dette jeg sette opp som vil ha effekt? Dette kutta en del sekunder i tid brukt på å bygge time-series, men det tar fortsatt ~25 sekunder for månedsvis fra 2023-01-01